README.md

```diff
@@ -26,31 +26,51 @@ Next follow these instructions.
     PASSWORD: osp
     HOST: localhost
     ``` 
-    You may run the following commands for local setup of DB:
+    You may run the following commands for local setup of DB in Linux:
 
     ```
+    cd open-source-programs-backend
     sudo -i -u postgres
     createuser osp --pwprompt
     psql
     CREATE DATABASE osp;
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
     ```
+2. you need to download API key file from your user-settings on Zulip. The file you download is named as 'download' or rename that to 'download'.
+3. Put that download file in the project's directory.
 
-
-2. To start the server:
+4. Move into the project's directory.
 
     ```
     cd open-source-programs-backend
+    ````
+5. You don't need to create virtual environment while working in Linux.
+    `````
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
+    ````
+6. To run the migrations run: 
+   ````
+   python manage.py migrate
+   ````
+7. To run the server:
+    ```
     python manage.py runserver
     ```
-3. Navigate to `http://localhost:8000/` in your browser.
-4. To change the port you may run `python manage.py runserver 0.0.0.0:<port_number>`
-5. To run the migrations run: `python manage.py migrate`
-6. You can terminate the process by `Ctrl+C` in your terminal.
+8. Navigate to `http://localhost:8000/` in your browser.
+9. To change the port you may run `python manage.py runserver 0.0.0.0:<port_number>`
+10. To run the migrations run: `python manage.py migrate`
+11. You can terminate the process by `Ctrl+C` in your terminal.
+
+Follow the given instructions for Login into the app.
+
+1. To create the superuser run:
```

Comment:
Although creating a superuser is an easy and fast method to login into the application. The correct way is to register a new user by setting up SendGrid details in the settings.py following the instructions from https://sendgrid.com/docs/for-developers/sending-email/integrating-with-the-smtp-api/ you can list this as the first step so that users can register a new user and then login into the application and if that doesn't work then they can create a superuser to login.